README.rst

```diff
@@ -271,102 +82,14 @@ Resources
     doi     = {10.1109/tpds.2021.3082815}
   }
 
-**Example Compatible Packages**
```

Comment:
As we have community examples, we should prob have a link to that.

Reply:
Community Examples is still linked to on line 67 on the right